packages/fxa-auth-server/fxa-oauth-server/lib/db/mysql/index.js

```diff
@@ -469,15 +472,17 @@ MysqlStore.prototype = {
     return this._write(QUERY_CODE_DELETE, [hash]);
   },
   generateAccessToken: function generateAccessToken(vals) {
+    // In certain scenarios to help avoid thundering herd, we add some padding to the expiration time of the token.
+    vals.ttl = (vals.ttl * 1000 || MAX_TTL) + EXPIRES_AT_TTL_PADDING;
```

Comment:
Should the second term in the sum be something like 

`(Math.floor(Math.random() * EXPIRES_AT_TTL_PADDING))`?

Reply:
Missed the most important part 🙇🏽‍♂️, fixed!